pkg/crc/machine/start.go

```diff
@@ -924,6 +924,13 @@ func logBundleDate(crcBundleMetadata *bundle.CrcBundleInfo) {
 }
 
 func ensureRoutesControllerIsRunning(sshRunner *crcssh.Runner, ocConfig oc.Config, preset crcPreset.Preset, bundleInfo *bundle.CrcBundleInfo) error {
+	// Check if the bundle have `/opt/crc/route_controller.yaml` file and if it has
+	// then use it to create the resource for router controller.
+	if _, _, err := sshRunner.Run("ls", "/opt/crc/route_controller.yaml"); err == nil {
+		_, _, err := ocConfig.RunOcCommand("apply", "-f", "/opt/crc/route_controller.yaml")
+		return err
+	}
+	// TODO: Remove this resource creation after crc 2.24 release
```

Comment:
Why 2.24? As a prerequisite for removing this, I'd first start with having a clear policy wrt bundles, and a clear error message that the bundle is not supported, not a cryptic failure.

Reply:
I can see you already created #3730 for that.